src/helpers/filter-by-field-function/FilterByField.js

```diff
@@ -0,0 +1,5 @@
+const FilterByField = (items, field, value) => {
+        return items.filter((item) => item[field] === value);
```

Comment:
could be just file helpers.js in here, no need in a separate folder and inside named export could be used